docs/source/developer/virtualdom.md

```diff
@@ -11,7 +11,7 @@ In situations where these feature are needed, we recommend using Phosphor's
 `Widget` class directly.
 
 The idea of virtual DOM rendering, which became popular in the
-[React](https://facebook.github.io/react/) community, is a very elegant and
+[React](https://nodejs.org/en/) community, is a very elegant and
```

Comment:
This link change seems strange.

Reply:
@vidartf Yeah, I just grabbed the redirect that facebook has currently from the old link as it displayed in `make linkcheck`. https://nodejs.org seems to also redirect to https://nodejs.org/en. Either is fine. Probably best to go with https://nodejs.org unless it errors in `make linkcheck`.

Reply:
The redirect in my browser goes to `https://reactjs.org/` not `https://nodejs.org/` ;)

Reply:
(at least currently, I'm not saying fb cannot have had an errant redirect 😅)

Reply:
You are completely right @vidartf. 👍 I was working too fast. This should absolutely be react 🤦‍♀️. 

@jasongrout @jzf2101 Can you fix this the next time you touch the docs?

Reply:
@vidartf already fixed it in https://github.com/jupyterlab/jupyterlab/pull/3667 :).

Reply:
@vidartf Thank you :sunny: